### DISCUSSION TITLE
[visits] Set-up specific Scanning Visits

### DISCUSSION THREAD

**Discussion by skarya22** [aces/Loris#9504] [General]
- CCNA would like to set-up scanning visits, similar to how sites have a scanning designation- other projects would likely benefit from this as well
- On CCNA we have it set where scanning visits can only take place at scanning sites, and scanning sites can only have scanning visits. This is to be discussed on whether this behaviour is desired for all projects
- The current thought process is to add a column to the visit table similar to psc, which designates the visit as a scanning visit

**Comment by ridz1208** [comment]
1. I don't think Sites have any "scanning designation" (AFAIK)
2. I like the Idea
3. it would be my feeling too but in the context of LORIS I think PSC needs a similar column too

**Reply by skarya22** [reply]
Sorry, misunderstood the MRI_Alias column for site designation

**Reply by ridz1208** [reply]
good point... not sure if that association is reinforced anywhere?

**Comment by ridz1208** [comment]
PS: we also may want to consider a Project level scanning flag? not sure exactly of the use cases

**Reply by skarya22** [reply]
We can consider it, but I am a bit confused as to what that would be used for?

**Reply by ridz1208** [reply]
I'm also not 100% on where it would not overlap with either visits or sites but in theory, a site is "scanning" when it has a scanner, a visit is "scanning" when a scan is aquired during the visit and a project would be "scanning" if they use or not imaging... So it might affect things like the dashboard you see or the tests list if you are affiliated with a scanning project

**Comment by CamilleBeau** [comment]
@ridz1208 Would it make sense to add the flag for all 3 of Sites, Project, & Visit levels? Currently we have this flag [on CCNA](https://github.com/aces/CCNA/blob/533691ee673131cc90d1b1ab4c14048ce8815cf7/tables_sql/Archive/Patch/2016_10_26_psc_AddMRISites.sql#L2) for sites

**Reply by ridz1208** [reply]
I would say yes but only if there is a usecase for it. right now I think we only have the usecase for visit and PSC

lets start with those

**Reply by christinerogers** [reply]
@skarya22 if there was any follow up on this discussion -- could you please post a link to the Issue / PR / Roadmaps item, and then close as resolved. 
(if there are no followup links, please don't close the discussion, just leave as-is. )
